armada/handlers/armada.py

```diff
@@ -162,12 +162,33 @@ def sync(self):
                                             chart.namespace,
                                             prefix,
                                             values=yaml.safe_dump(values))
-
+	
             LOG.debug("Cleaning up chart source in %s",
                       chartbuilder.source_directory)
 
             chartbuilder.source_cleanup()
 
+        #
+        # TODO: wait until all charts are in status=Running
+        #
+        os_pods = self.tiller.k8s.get_namespace_pod("openstack")
```

Comment:
so this will make sure that openstack is the one we look at and validate that its running, but we in a armada yaml we could have multiple namespaces. we need to check all namespaces in the yaml are looked at.